products/ubuntu2204/profiles/stig.profile

```diff
@@ -26,7 +26,7 @@ selections:
     # UBTU-22-612040 The Ubuntu operating system must map the authenticated identity to the user or group account for PKI-based authentication.
     - verify_use_mappers
 
-    # UBTU-22-411025 The Ubuntu operating system must enforce 24 hours/1 day as the minimum password lifetime. Passwords for new users must have a 24 hours/1 day minimum password lifetime restriction.
+    # UBTU-22-411025 The Ubuntu operating system must enforce 24 hours/one day as the minimum password lifetime. Passwords for new users must have a 24 hours/one day minimum password lifetime restriction.
```

Comment:
on line 3, please update the version too as  well as adding the manual to shared/references.

Reply:
Done